README.md

```diff
@@ -6,8 +6,13 @@ CoreTools Healthcheck: [![CoreTools healthcheck](https://azure-functions.visuals
 
 ## V4 Images
 
+All languages have a default version published as the major version tag without a language version suffix. Suggested usage is to define the language version using the tags below.  There are no guarantees that the default language version will be static. 
+ Ex. `node:4-appservice` -> `node:4-node14-appservice` (preferred)
```

Comment:
instead of this - lets remove tags that do not have explicit language version from the readme

Reply:
All of them have been removed other than dotnet.  Cut the highlight portion as well